frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.test.ts

```diff
@@ -0,0 +1,89 @@
+import { PackagesRouter } from './PackagesRouter';
+
+const mockOrg: string = 'org';
+const mockApp: string = 'app';
+
+describe('PackagesRouter', () => {
+  describe('constructor', () => {
+    it('should initialize app and org properties', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+
+      expect(packagesRouter['app']).toEqual(mockApp);
+      expect(packagesRouter['org']).toEqual(mockOrg);
```

Comment:
Hmm, did you use `packagesRouter["app"]` to access private variables? 🤔 Do we need a separate test for this? We should ensure that the public method, which returns the URL, includes the parameters (org, app). By doing so, we can confirm that our constructor works as expected, then we can delete this constructor test.

Reply:
I agree that it might not be needed 😄 It is removed now